defaults/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# NB. Supported RHEL versions: RHEL 8+
+#
+# Enable or disable service after configuration
+# to allow testing and verification before use.
+# NB. See ima_evm_setup if planning to use IMA.
+fapolicyd_setup_enable_service: false
+
+# trust list for fapolicyd configuration file
+# default "rpmdb,file"
+fapolicyd_setup_trust: "{{ '' if ansible_facts.distribution_version is
+                          version('8.2', '<=') else 'rpmdb,file' }}"
+
+# set integrity
+# default "none"
+# can be "none", "size", "sha256", "ima"
+# in case of ima, kernel's IMA has to be setup correctly
+fapolicyd_setup_integrity: "{{ '' if ansible_facts.distribution_version is
+                          version('8.3', '<=') else 'none' }}"
+
+# set permissive mode
+fapolicyd_setup_permissive: false
+
+# fapolicyd trust file managament
+# list of trusted files
+
+fapolicyd_add_trusted_file: "{{ '' if ansible_facts.distribution_version is
+                            version('8.2', '<=') else [] }}"
```

Comment:
```suggestion
                            version('8.2', '<=') and ansible_facts.os_family == 'RedHat'
                            else [] }}"
```

Reply:
`We don't need to check this because correct distro/version is checked in System check.`

Also applies here.